router/terraform/cloudfront.tf

```diff
@@ -69,6 +69,16 @@ resource "aws_cloudfront_distribution" "wellcomecollection_org" {
         ]
       }
     }
+
+    # lambda_function_association {
```

Comment:
Is this coming back at some point?

Reply:
Yip, once the code looks good to people, thought it might be good to pair adding it back, not that the code is hard, but the thinking would be good to run through, and see if we can figure out if it feels safe enough.